Plugin~/WebRTCPlugin/GpuMemoryBuffer.cpp

```diff
@@ -83,7 +83,9 @@ namespace webrtc
         if (!device_->WaitSync(textureCpuRead_.get(), timeout.count()))
         {
             RTC_LOG(LS_INFO) << "WaitSync failed.";
-            return nullptr;
+
+            // returns buffer because the video encoders don't check nullptr.
+            return I420Buffer::Create(textureCpuRead_->GetWidth(), textureCpuRead_->GetHeight());
```

Comment:
Curious, this is an Android workaround? Is it against Android specific encoder or in general, does it happen often?

I don't have an Android device to test against, but an earlier iteration of the code did work there during testing, but it doesn't say much because there was no WaitSync implementation at the time.

Reply:
The program passes through this line on Android device very often.
Maybe this crash is not only Android encoder, if the device performance is very low, waitsync method is failed because of the timeout.

Reply:
In my experience encoding without nvenc is pretty slow with vulkan, but I should have a small follow-up to this PR that should help out